client/src/pages/Search/GridView/RestaurantCard/ImageCard.js

```diff
@@ -7,7 +7,15 @@ const classes = {
 };
 
 function ImageCard({ src }) {
-  return <img src={src} style={classes.imageCard} alt={src} />;
+  return <img 
+  src={src}
+  style={classes.imageCard}
+  onError={({ currentTarget }) => {
+    currentTarget.onerror = null; // prevents looping
+    currentTarget.src="https://images.unsplash.com/photo-1546069901-ba9599a7e63c?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8MXx8fGVufDB8fHx8&w=1000&q=80";
```

Comment:
Not a fan of how this was implemented.
You could just have something like:
```jsx
src = { restaurant_image ?? fallback_image }
```
Ask @JoshuaRamc about value coalescing 

Also, don't like how the default picture is a colorful chicken salad. Doesn't give the impression that the restaurant has no image. Use a bland image like this instead:
![image](https://user-images.githubusercontent.com/51217000/230692533-1b657ec9-1a95-42a6-98f0-673dbc35ae85.png)

Reply:
using nullish coalescing lets you remove that `onError` prop

Reply:
hint: This file should only have 2 line changes

import defaultImage
nullish coalesce on image url